bridge-adaptivity/bridge_lti/provider.py

```diff
@@ -26,14 +26,40 @@ def lti_launch(request, collection_id=None):
     try:
         lti_consumer = LtiProvider.objects.get(consumer_key=params['oauth_consumer_key'])
     except LtiProvider.DoesNotExist:
-        return HttpResponseForbidden()
+        # wrong 'consumer_key':
+        return render(
+            request,
+            template_name="bridge_lti/announcement.html",
+            context={
+                'title': 'forbidden',
+                'message': 'please, check provided LTI credentials.',
+                'tip': 'have a look at `consumer key`',
+            }
+        )
 
     if not SignatureValidator(lti_consumer).verify(request):
-        return HttpResponseForbidden()
+        # wrong 'consumer_secret':
+        return render(
+            request,
+            template_name="bridge_lti/announcement.html",
+            context={
+                'title': 'forbidden',
+                'message': 'please, check provided LTI credentials.',
+                'tip': 'have a look at `consumer secret`',
+            }
+        )
 
     if params['roles'] in ['Student', 'Learner']:
```

Comment:
This statement should be rewritten params['roles'] returns string with the roles separated with the comma so you could get `params['roles'] == 'Instructor,Administrator, Mentor'` 

Please take a look at docs: https://www.imsglobal.org/specs/ltiv1p1/implementation-guide#toc-11

And change statement to the opposite check if role is Instructor otherwise all roles are proposed student view

Reply:
This will be addressed in the #19

Reply:
Got it thanks